src/bindings/js/node/tests/basic.test.js

```diff
@@ -20,6 +20,13 @@ it('Core.getAvailableDevices()', () => {
     assert.ok(devices.includes('CPU'));
 });
 
+it('Core.getVersions()', () => {    
+  const devices = core.getAvailableDevices();
+  const deviceVersion = core.getVersions(devices[0]);
+
+  assert.ok(deviceVersion.size()>=1);
+});
```

Comment:
Tests have to check behavior in the cases when incorrect parameters passed, parameters are absent and that result has expected format.